### ISSUE TITLE
commit [1c8701f] fails to compile

### ISSUE THREAD

**Issue by EastAPOLO** [too-r/lambdaOS#15]
```
stefan:[~/sources/lambdaOS]$ make -j10
make: *** No rule to make target 'target/x86_64-lambda/debug/liblambdaOS.a', needed by 'build/kernel-x86_64.bin'.  Stop.
make: *** Waiting for unfinished jobs....
   Compiling bit_field v0.7.0
   Compiling once v0.3.3
   Compiling bitflags v1.0.1
   Compiling siphasher v0.2.2
   Compiling spin v0.4.6
   Compiling volatile v0.1.0
   Compiling byteorder v0.5.3
error: Error loading target specification: Could not find specification for target "x86_64-lambda"
  |
  = help: Use `--print target-list` for a list of built-in targets

error: Error loading target specification: Could not find specification for target "x86_64-lambda"
  |
  = help: Use `--print target-list` for a list of built-in targets

error: Error loading target specification: Could not find specification for target "x86_64-lambda"
  |
  = help: Use `--print target-list` for a list of built-in targets

error: Error loading target specification: Could not find specification for target "x86_64-lambda"
  |
  = help: Use `--print target-list` for a list of built-in targets

error: Could not compile `once`.
warning: build failed, waiting for other jobs to finish...
error: Could not compile `bitflags`.
warning: build failed, waiting for other jobs to finish...
error: Could not compile `siphasher`.
warning: build failed, waiting for other jobs to finish...
error: Error loading target specification: Could not find specification for target "x86_64-lambda"
  |
  = help: Use `--print target-list` for a list of built-in targets

error: Could not compile `bit_field`.
warning: build failed, waiting for other jobs to finish...
error: Error loading target specification: Could not find specification for target "x86_64-lambda"
  |
  = help: Use `--print target-list` for a list of built-in targets

error: Could not compile `volatile`.
warning: build failed, waiting for other jobs to finish...
error: Could not compile `spin`.
warning: build failed, waiting for other jobs to finish...
error: build failed
make: *** [Makefile:49: kernel] Error 101
```

**Comment by robert-w-gries** [comment]
See related issue phil-opp/blog_os#379.

I committed a fix in phil-opp/blog_os#381

**Comment by too-r** [comment]
@robert-w-gries Thanks, you beat me to it. I've been meaning to get around to setting this in the Makefile, but I guess since people other than me are actually trying to build this now, I might as well fix it sooner.
